unitTests/ui/DidYouMeanTest.ts

```diff
@@ -65,61 +65,89 @@ export function DidYouMeanTest() {
       expect(analyticsSpy).toHaveBeenCalledWith(analyticsActionCauseList.didyoumeanClick, {});
     });
 
-    describe('exposes options', function() {
-      describe('enableAutoCorrection', function() {
-        it('set to true should autocorrect the query when no results are found', function() {
-          test = Mock.optionsComponentSetup<DidYouMean, IDidYouMeanOptions>(DidYouMean, <IDidYouMeanOptions>{
-            enableAutoCorrection: true
+    describe('exposes options', () => {
+      describe('enableAutoCorrection', () => {
+        describe('set to true', () => {
+          beforeEach(() => {
+            test = Mock.optionsComponentSetup<DidYouMean, IDidYouMeanOptions>(DidYouMean, <IDidYouMeanOptions>{
+              enableAutoCorrection: true
+            });
+            test.env.queryStateModel.get = () => 'foobar';
           });
-          test.env.queryStateModel.get = () => 'foobar';
-          var qsmSpy = jasmine.createSpy('queryStateModelSpy');
-          test.env.queryStateModel.set = qsmSpy;
 
-          Simulate.query(test.env, {
-            results: FakeResults.createFakeResults(0),
-            queryCorrections: [fakeQueryCorrection]
-          });
+          it('should autocorrect the query when no results are found', () => {
+            const qsmSpy = jasmine.createSpy('queryStateModelSpy');
```

Comment:
I think it is better to not abbreviate the variable name.

Reply:
Good point, I was merely copy pasting the old tests, but I can take the time to rename them.